tex/main.tex

```diff
@@ -28,6 +28,45 @@ \section{Introduction}\label{sec:introduction}
     \item How do 1 and 2 change as a function of population size?
 \end{enumerate}
 
+A key point here is that the relative fitness of a strategy depends on the population distribution. The original Moran process assumes a relative fitness of r of one strategy over the other, giving a fixation probability for the starting population $(i, N-i)$ (when $r != 1$)
+
+\[ \rho = \frac{1 - r^{-i}}{1 - r^{-N}}\]
+and $\rho = 1 / N$ if $r=1$ (the neutral fixation probability).
+
+This corresponds to a game matrix [[1, 1], [r, r]] (or [[r, r], [1, 1]]), which is of course not what we have -- it's a little complicated because our "fitness" is not the payout from the game matrix, rather the sum of the total scores of all the interactions each round. So ALLC and TFT are neutral wrt to each other because they will have the same score each round, giving an effective fitness landscape $f(i, N-i) = A [i, N-i]^T$ given by the matrix $A = [[1,1],[1,1]]$. This means that noise and the number of turns per Moran round are significant parameters. I think we should fix the turns at 200; some recent authors run the turns to infinity (to reach stationarity on the sub-"Markov process" on the states (C, C), (C, D), (D, C), (D, D)) but we can't analytically compute the stationary distribution for strategies that use more than one round of memory (and it's not really a Markov process for more than one round of memory anyway). Plus it's unrealistic, and ultimately just amounts to a transform of the game matrix.
+
+To see if one strategy is not neutral with respect to another, we want to empirically measure the fixation probability and compare to the neutral rate. To do this right we need a lot of counts, since we're estimating a binomial probability p with variance $p(1-p) / k$ and $p$ is close to $1 / N$. To get the variance small you need something like $k>1000$ observations ( we can work out the precise requirements).
```

Comment:
Currently running with 1000 repetitions. Could potentially double that, once we see how long it takes to run etc.